topics/genome-annotation/tutorials/funannotate/tutorial.md

```diff
@@ -39,6 +39,9 @@ contributions:
     - eurosciencegateway
     - elixir-europe
 
+edam_ontology:
+  - topic_0622 # Genomics
```

Comment:
genomics at least can probably go at the topic level.

Reply:
Thanks @hexylena!